### ISSUE TITLE
Missing info on new rule in 3.3.0

### ISSUE THREAD

**Issue by bjankord** [dequelabs/axe-core#1683]
We are using axe-core 3.3.0 and started to see new error output with this release.

```
expected no accessibility violations but got:
        {
  "description": "Elements that have scrollable content should be accessible by keyboard",
  "help": "Ensure that scrollable region has keyboard access",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.3/scrollable-region-focusable?application=axeAPI",
  "id": "scrollable-region-focusable",
  "impact": "moderate",
  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": null,
          "id": "focusable-content",
          "impact": "moderate",
          "message": "Element should have focusable content",
          "relatedNodes": []
        },
        {
          "data": null,
          "id": "focusable-element",
          "impact": "moderate",
          "message": "Element should be focusable",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  Element should have focusable content\n  Element should be focusable",
      "html": "<div class=\"PopupOverlay-module__popup-overlay___Dsi-h\"><div class=\"PopupOverlay-module__inner___3zbLr\"></div></div>",
      "impact": "moderate",
      "none": [],
      "target": [
        ".PopupOverlay-module__popup-overlay___Dsi-h"
      ]
    }
  ],
  "tags": [
    "wcag2a",
    "wcag211"
  ]
}
```

When navigating to the help URL, the page doesn't provide much help in terms of how one might solve this error.

https://dequeuniversity.com/rules/axe/3.3/scrollable-region-focusable?application=axeAPI

<img width="971" alt="Screen Shot 2019-07-08 at 3 14 28 PM" src="https://user-images.githubusercontent.com/633148/60839639-41571000-a193-11e9-94c0-af5c48de01b4.png">

**Comment by straker** [comment]
Thanks for reporting this. We'll get our documentation sorted out as soon as we can.

**Comment by straker** [comment]
The docs have been updated with all the relevant information.
